packages/frontend/src/components/sign/SignTransferMultipleAccounts.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { Translate } from 'react-localize-redux';
+import styled from 'styled-components';
+
+import ArrowUpImage from '../../images/icon-arrow-up-green.svg';
```

Comment:
Can probably remove the `ArrowUpImage` import too?

Reply:
👍